docs/content/users/quickstart.md

```diff
@@ -192,6 +192,21 @@ ddev launch
 
     When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
 
+## Ibexa DXP
+
+Install Ibexa DXP OSS Edition. Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases.
```

Comment:
Can move "Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases." sentence to the end of the section :wink:

Reply:
Done